docs/docs/en/guide/task/stored-procedure.md

```diff
@@ -8,6 +8,32 @@
    <img src="/img/procedure-en.png" width="80%" />
  </p>
 
-- DataSource: The DataSource type of the stored procedure supports MySQL and POSTGRESQL, select the corresponding DataSource.
-- Method: The method name of the stored procedure.
-- Custom parameters: The custom parameter types of the stored procedure support `IN` and `OUT`, and the data types support: VARCHAR, INTEGER, LONG, FLOAT, DOUBLE, DATE, TIME, TIMESTAMP and BOOLEAN.
\ No newline at end of file
+- Node name: The node name in a workflow definition is unique.
```

Comment:
you should change the document like https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/guide/task/sql.md. We have section `Overview ` and section `Create DataSource`

Reply:
https://raw.githubusercontent.com/apache/dolphinscheduler/dev/docs/docs/en/guide/task/sql.md @hstdream